src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java

```diff
@@ -54,6 +49,20 @@ public void check(Network network) throws NetworkModificationException {
 
         checkConverterStation(network, modificationInfos.getConverterStation1());
         checkConverterStation(network, modificationInfos.getConverterStation2());
+        checkDroop();
+    }
+
+    private void checkDroop() {
+        // extension is not enabled => ignore check inside fields
+        if (modificationInfos.getAngleDroopActivePowerControl() == null || !modificationInfos.getAngleDroopActivePowerControl()) {
+            return;
```

Comment:
These modification infos will be ignored if the control is not activated. We can show an warning but not here because the implementation of check method is for a blocking issue, in other word we throw an exception that automatically captured and then created an error report.

For the simplicity, I do not implement an warning when do not create an extension but Droop or P0 provided.

However, I modified the check as yours comment in others places:
```java
        // extension is not enabled => ignore check inside fields
        if (!Boolean.TRUE.equals(modificationInfos.getAngleDroopActivePowerControl())) {
            return;
        }
```